test/createRoom-test.ts

```diff
@@ -109,6 +109,22 @@ describe("createRoom", () => {
         expect(createJitsiCallSpy).not.toHaveBeenCalled();
         expect(createElementCallSpy).not.toHaveBeenCalled();
     });
+
+    it("correctly sets up MSC3401 power levels", async () => {
+        await createRoom({});
+
+        const [[{
+            power_level_content_override: {
+                events: {
+                    [ElementCall.CALL_EVENT_TYPE.name]: callPower,
+                    [ElementCall.MEMBER_EVENT_TYPE.name]: callMemberPower,
+                },
+            },
+        }]] = client.createRoom.mock.calls as any; // no good type
```

Comment:
what was the error it was giving you? Casting before splitting off pieces of use is a bit surprising.

Reply:
`Property 'events' does not exist on type '{}'.`. `power_level_content_override` is typed as `object` in the js-sdk which doesn't seem to be doing the right thing here?

Reply:
The types in the js-sdk should probably be fixed, then. `object` is insufficient :)